prototype/technology.lua

```diff
@@ -1357,11 +1354,11 @@ data:extend(
                 unit = {
                     count = 500,
                     ingredients = {
-                        { "automation-science-pack", 1 },
-                        { "logistic-science-pack", 1 },
+                        { "automation-science-pack", 2 },
+                        { "logistic-science-pack", 2 },
```

Comment:
Why were those changed to 2?

Reply:
have u seen
![image](https://user-images.githubusercontent.com/59801736/165779005-c7a04388-b5ce-418f-82ab-52ace4c64a8b.png)

Reply:
Yes, obviously they have been changed cause you changed them, the question is why you changed them. They should not have changed.

Reply:
Well then do it all yourself. Don't ask me to add PR if u just are being a dick about it.